Taskfile.yml

```diff
@@ -277,6 +277,13 @@ tasks:
     cmds:
       - "{{.GENERATOR_ROOT}}bin/{{.GENERATOR_APP}} gen-kustomize {{.CONTROLLER_ROOT}}config/crd"
 
+  controller:run-kustomize:
+    desc: Generates the CRD & configuration bundle.
+    deps: [controller:generate-kustomize]
+    dir: "{{.CONTROLLER_ROOT}}"
+    cmds:
+    - kustomize build config/default | sed -e 's@localhost:5000/azureserviceoperator:latest@{{.CONTROLLER_DOCKER_IMAGE}}@g' > bin/azureserviceoperator_{{.VERSION}}.yaml
```

Comment:
Do we want the file to have a `VERSION` suffix? I am not sure if there is a pattern for exactly how this is done (I looked across a number of repos and honestly I don't see one).

As discussed on teams this is fine.